civicrm_resources/civicrm_resources.module

```diff
@@ -59,6 +61,46 @@ function civicrm_resources_add_resources($extention_path, $req_files = []) {
   }
 }
 
+/**
+ * Filters files into css or js
+ *
+ * @param array $files
+ * @return array
+ */
+function filter_files($files) {
+  $filteredJsFile = array_filter($files, function ($file) {
+    return preg_match('/\.js$/', $file->filename);
+  });
+  $filteredCssFile = array_filter($files, function ($file) {
+    return preg_match('/\.css$/', $file->filename);
+  });
+  return [$filteredJsFile, $filteredCssFile];
```

Comment:
Please add some spaces here between the function calls to improve the readbility